slic3r.pl

```diff
@@ -6,7 +6,7 @@
 BEGIN {
     use FindBin;
     use lib "$FindBin::Bin/lib";
-    use local::lib '--no-create', "$FindBin::Bin/local-lib";
```

Comment:
Did this file actually change? If not, please commit a reversion (probably a manual one) and leave your local patch uncommitted.